workspaces/ui-v2/src/entry-points/local/local-cli.tsx

```diff
@@ -151,7 +151,7 @@ export function useLocalCliServices(
     baseUrl: apiBaseUrl,
     spectacle,
   });
-  const specRepository = new LocalCliSpecRepository({ baseUrl: apiBaseUrl });
+  const specRepository = new UILocalCliSpecRepository({ baseUrl: apiBaseUrl });
```

Comment:
There's two localcli spec repositories (one that runs in the cli, one that runs in memory in the UI)